index.html

```diff
@@ -5,12 +5,81 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&amp;lang=en" />
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" type="text/css" href="style.css" />
+
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <nav class="main-navigation">
+          <ul class="menu">
```

Comment:
This looks complicated - I would recommend starting with a single navigation menu (rather than a `menu` and a `sub-menu`).

If you want to make it responsive, you can start with something simple, such as having the elements align vertically or horizontally.